src/js/11-search-ui.js

```diff
@@ -0,0 +1,22 @@
+if (window.location.host !== 'developers.plentymarkets.com') {
```

Comment:
Which other hosts are possible here and why do they behave different?

Reply:
The host is either developers or knowledge. Developers still uses Algolia as search engine, which comes with its own UI. This will be cleaned up when switching to Elasticsearch on developers with #117.